components/applications-service/integration_test/suite_test.go

```diff
@@ -66,6 +71,17 @@ func NewSuite(database string) *Suite {
 	// ```
 	s.StorageClient = dbClient
 
+	// A global Ingester instance to ingest any "message" into our system
+	//
+	// From any test you can directly call:
+	// ```
+	// res, err := suite.Ingester.IgestMessage(msg)
+	// ```
+	s.Ingester = ingest.New(c, s.StorageClient)
+
+	// Start processing messages as they are sended to the ingest events channel
```

Comment:
`s/sended/sent/` If the spellchecker doesn't complain now, it could in the future.

Reply:
Changed! Thanks